alibi_detect/od/sr.py

```diff
@@ -1,24 +1,25 @@
+import enum
 import logging
 import numpy as np
-from typing import Dict
+from typing import Dict, Literal
```

Comment:
Small thinkg I forgot, `Literal` is not in the standard lib for Python 3.6 so for support would need to do something like this:
https://github.com/SeldonIO/alibi/blob/0ccf1b726b6a865b5a94d727847653ffc7e68a4f/alibi/explainers/ale.py#L11-L14

We will drop Python 3.6 support very soon though, though probably not for the next release so best to put this in. @ascillitoe

Reply:
Actually it is only present in standard lib for Python 3.8+ so we won't be able to get rid of the version check for a while as 3.7 will be supported for some time.